app/assets/stylesheets/_forms.scss

```diff
@@ -1,3 +1,15 @@
+.elements-error-summary {
+  border: 5px solid #b10e1e;
```

Comment:
@Davidslv The colour variables come directly from Bootstrap.
`govuk_admin_template` defines how we should use Bootstrap and is responsible for keeping the look and feel consistent between apps.

Red in Bootstrap is one colour, red in toolkit is another. This is the first example where we are using elements over Bootstrap, hence the slight weirdness.